website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,41 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
```

Comment:
The `isRounded` argument is missing from this list.

Reply:
Added in [0dca0f1](https://github.com/hashicorp/design-system/pull/1749/commits/0dca0f10b9926f99f7e51399f1b723ca86a8a6bf) – now named `isStandalone`, as agreed